ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/manager/EthServerTest.java

```diff
@@ -58,32 +58,24 @@ public void setUp() {
         new EthProtocolConfiguration(2, 2, 2, 2, 2, false));
   }
 
-  @Test
-  public void shouldRespondToNodeDataRequests() throws Exception {
-    when(worldStateArchive.getNodeData(HASH1)).thenReturn(Optional.of(VALUE1));
-    when(worldStateArchive.getNodeData(HASH2)).thenReturn(Optional.of(VALUE2));
-    ethMessages.dispatch(new EthMessage(ethPeer, GetNodeDataMessage.create(asList(HASH1, HASH2))));
-
-    verify(ethPeer).send(NodeDataMessage.create(asList(VALUE1, VALUE2)));
-  }
-
```

Comment:
`ethMessages` is no longer responsible for `send`ing. Also, this behavior is already covered by `EthProtocolManagerTest`